tests/e2e/wilcoEngine/wilcoEngineEvents.js

```diff
@@ -0,0 +1,13 @@
+const dispatch = (type) => {
```

Comment:
Interesting approach. Can you explain why did you chose to use the global object?
I understand you want to make it in the available in the global scope. Just wondering why no using it's own class and then import it

Reply:
It just won't work. We want to have one server module and not run a server for each file. And it does not matter how the test will interact with the sever (emitter, knocked functions, etc) the solution need to be global. Otherwise the server module and the tests will use different instances

Reply:
I see, I'd assume it will behave the same as the server which is initiated only once in the jest setup? Anyways approach is very easy to read which is cool